pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -29,7 +29,7 @@ import (
 	secretsmanager "github.com/gardener/gardener/pkg/utils/secrets/manager"
 
 	"github.com/gardener/gardener/pkg/resourcemanager/controller/garbagecollector/references"
```

Comment:
nit:
can you move `github.com/gardener/gardener/pkg/resourcemanager/controller/garbagecollector/references` to the upper section of imports?

Reply:
Just to get it right: Do you want me to add a completely random change to this pull request? This does not feel right to me.
I can open a separate pull request for this (but so can you). Seems like this was introduced in #4518 and then changed in #4757.
What do you think?

Reply:
Just a suggestion for a clean up because it was close to code that was edited in this PR (line 32 where "google.golang.org/protobuf/types/known/durationpb" is added). 

Generally we try to keep packages from gardener/gardener in the same import section. It was introduced in https://github.com/gardener/gardener/pull/4518, but back then the gardener resource manager was not part of the gardener/garedener repository.

Reply:
Created separate pull request https://github.com/gardener/gardener/pull/6315 for this.

Reply:
No strong opinions, but maybe we can do it as a separate commit in this PR?

Reply:
From my perspective, this is a pretty random clean up, which has nothing to do with the istio update. Therefore, it ist not justified to be in the same pull request.

I prefer clear separation of topics. "Smuggling" in changes make it harder to understand afterwards why a certain change was done as part of a certain pull request.